src/OpenTelemetry.Exporter.Prometheus.Shared/.publicApi/net462/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,6 @@
+OpenTelemetry.Exporter.Prometheus.Shared.PrometheusExporterOptions
```

Comment:
We don't need to run publicApi analyzer for the shared project. This might require adding some new condition in `Common.prod.props`. Currently, publicApi analyzer is run for every project under `src/`.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3497